cli-plugins/apps/reference/workload-create-apply.hbs.md

```diff
@@ -15,7 +18,7 @@ shown in YAML file format.
 Example
 
 ```console
-tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.5.0 --type web
+tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.6.0 --type web
```

Comment:
assuming a git-tag will be created for every TAP release, you can replace all references to the specific TAP version (e.g. `1.6.0`) with the value from `/template_variables.yaml` as follows:

tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-{{ vars.tap_version }} --type web

There probably a large number of references in your docs updates overall so you should do a broad scrub to apply the variable across the board.

Also - if you believe it would make things more efficient - you should consider adding another template variable for the apps plugin version (if it's referenced frequently through docs).

And it's worth noting the other variables already there that you can use, for the tanzu cli, etc....
Enjoy!

Reply:
are the code block sections going to take the change as well?

Reply:
used the suggested way but not sure if the code blocks will be taking the version

@kcoriordan for confirmation